src/implementations/twig/components/footer-standardised/footer-standardised.html.twig

```diff
@@ -49,103 +50,94 @@
-    {% endif %}
-    {# logo (EU) #}
-    {% if _section.logo is defined and _section.logo is not empty %}
-      {% set _label %}
-        <img
-          alt="{{ _section.logo.alt }}"
-          title="{{ _section.logo.title }}"
-          class="ecl-footer-standardised__logo-image-mobile"
-          src="{{ _section.logo.src_mobile }}"
-        />
-        <img
-          alt="{{ _section.logo.alt }}"
-          title="{{ _section.logo.title }}"
-          class="ecl-footer-standardised__logo-image-desktop"
-          src="{{ _section.logo.src_desktop }}"
-        />
-      {% endset %}
-      {% include '@ecl/link/link.html.twig' with {
-        link: {
-          path: _section.logo.path,
-          label: _label,
-          type: 'standalone',
-          aria_label: _section.logo.title
-        },
-        extra_classes: 'ecl-footer-standardised__logo-link'
-      } only %}
-    {% endif %}
-    {# content before #}
-    {% if _section.content_before is defined and _section.content_before is not empty %}
-      <div class="ecl-footer-standardised__content">{{ _section.content_before }}</div>
-    {% endif %}
-    {# title #}
-    {% if _section.title is defined and _section.title is not empty %}
-      {% set _title_class_name = 'ecl-footer-standardised__title' %}
-      {% if _section.title_class_name is defined and _section.title_class_name is not empty %}
-        {% set _title_class_name = _title_class_name ~ ' ' ~ _section.title_class_name %}
-      {% endif %}
-      {% if _section.title.link is defined and _section.title.link is not empty %}
-        {% include '@ecl/link/link.html.twig' with _section|merge({
-          link: _section.title.link|merge({ type: 'standalone'}),
-          extra_classes: _title_class_name,
-          icon: _section.title.icon|default({}),
-        }) only %}
-      {% else %}
-        <div class="{{ _title_class_name }}">
-          {{- _section.title -}}
+  {% for _row in _rows %}
+    <div class="ecl-footer-standardised__row">
+    {% for _columns in _row %}
+      <div class="ecl-footer-standardised__column">
+      {% for _sections in _columns %}
+        {% set _section_class_name = 'ecl-footer-standardised__section' %}
+        {% if _sections.section_class_name is defined and _sections.section_class_name is not empty %}
+          {% set _section_class_name = _section_class_name ~ ' ' ~ _sections.section_class_name %}
+        {% endif %}
+        <div class="{{ _section_class_name }}">
+          {# logo (EU) #}
+          {% if _sections.logo is defined and _sections.logo is not empty %}
+            {% set _label %}
+              <img
+                alt="{{ _sections.logo.alt }}"
+                title="{{ _sections.logo.title }}"
+                class="ecl-footer-standardised__logo-image-mobile"
+                src="{{ _sections.logo.src_mobile }}"
+              />
+              <img
+                alt="{{ _sections.logo.alt }}"
```

Comment:
Well, we never did it before, but better start, we will have to do it massively on all the templates, this is an attribute, let's escape it properly with `|e('html_attr')`